.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
 jobs = auto 
-max-line-length = 120 
```

Comment:
Why was this changed?

Reply:
To be in line with the settings in Black formatter file. You're right, it is better to change it the other way around...